lib/api/dsl/storage/index.js

```diff
@@ -35,17 +33,19 @@ const
   FieldOperand = require('./objects/fieldOperand'),
   containsOne = require('../util/containsOne');
 
+let
+  _hash;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the declaration of "_hash" to line 45. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2392)